pkgs/by-name/ku/kubectl-slice/package.nix

```
@@ -0,0 +1,28 @@
+{
+  lib,
+  buildGo124Module,
+  fetchFromGitHub,
+}:
+
+buildGo124Module rec {
+  pname = "kubectl-slice";
+  version = "1.4.2";
+
+  src = fetchFromGitHub {
+    owner = "patrickdappollonio";
+    repo = pname;
+    tag = "v${version}";
+    hash = "sha256-C9YxMP9MCKJXh3wQ1JoilpzI3nIH3LnsTeVPMzri5h8=";
+  };
+
+  vendorHash = "sha256-Lly8gGLkpBAT+h1TJNkt39b5CCrn7xuVqrOjl7RWX7w=";
+
+  meta = {
+    description = "Split multiple Kubernetes files into smaller files with ease.";
```

```suggestion
    description = "Split multiple Kubernetes files into smaller files with ease";
```
Not end with a period (or any punctuation for that matter).